package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -33,1527 +33,17 @@
-
-        self.box = self.u.dimensions if self.pbc else None
-        self._residue_dict = {}
-        self._build_residue_dict(self.selection1)
-        self._build_residue_dict(self.selection2)
-        self._build_residue_dict(self.water_selection)
-
-        self._update_selection()
-
-        self.timesteps = []
-        if len(self._s1) and len(self._s2):
-            self._update_water_selection()
-        else:
-            logger.info("WaterBridgeAnalysis: no atoms found in the selection.")
-
-        logger.info("WaterBridgeAnalysis: initial checks passed.")
-
-        logger.info("WaterBridgeAnalysis: starting")
-        logger.debug("WaterBridgeAnalysis: donors    %r", self.donors)
-        logger.debug("WaterBridgeAnalysis: acceptors %r", self.acceptors)
-        logger.debug("WaterBridgeAnalysis: water bridge %r", self.water_selection)
-
-        if self.debug:
-            logger.debug("Toggling debug to %r", self.debug)
-        else:
-            logger.debug("WaterBridgeAnalysis: For full step-by-step debugging output use debug=True")
-
-        logger.info("Starting analysis (frame index start=%d stop=%d, step=%d)",
-                    self.start, self.stop, self.step)
-
-    def _donor2acceptor(self, donors, h_donors, acceptor):
-        if len(donors) == 0 or len(acceptor) == 0:
-            return []
-        if self.distance_type != 'heavy':
-            donors_idx = list(h_donors.keys())
-        else:
-            donors_idx = list(donors.keys())
-        result = []
-        # Code modified from p-j-smith
-        pairs, distances = capped_distance(self.u.atoms[donors_idx].positions,
-                                           self.u.atoms[acceptor].positions,
-                                           max_cutoff=self.distance, box=self.box,
-                                           return_distances=True)
-        if self.distance_type != 'heavy':
-            tmp_donors = [h_donors[donors_idx[idx]] for idx in pairs[:,0]]
-            tmp_hydrogens = [donors_idx[idx] for idx in pairs[:,0]]
-            tmp_acceptors = [acceptor[idx] for idx in pairs[:,1]]
-        else:
-            tmp_donors = []
-            tmp_hydrogens = []
-            tmp_acceptors = []
-            for idx in range(len(pairs[:,0])):
-                for h in donors[donors_idx[pairs[idx,0]]]:
-                    tmp_donors.append(donors_idx[pairs[idx,0]])
-                    tmp_hydrogens.append(h)
-                    tmp_acceptors.append(acceptor[pairs[idx,1]])
-
-        angles = np.rad2deg(
-            calc_angles(
-                self.u.atoms[tmp_donors].positions,
-                self.u.atoms[tmp_hydrogens].positions,
-                self.u.atoms[tmp_acceptors].positions,
-                box=self.box
-            )
+warnings.warn(
+            "This module has been moved to MDAnalysis.analysis.hydrogenbonds"
+            "It will be removed in MDAnalysis version 2.0. Please use "
```

Comment:
Next target version is 2.0, so removals have to be done afterwards unless we want to make this a v1.1 target instead. Thoughts @orbeckst ?

Reply:
Are we doing more versions of 1? I would deprecate in 1.xx and remove in 2.0, as was done for #2791 and #2622.

Reply:
If we have to do a 1.1 for whatever reasons then we keep things as unchanging as possible. But the goal was to keep 1.x boring and as similar to the old stuff as possible, so any removals target 2.0.

I _hope_ that we are only doing bug fixes for 1.x so that this stays 1.0.x